### PR TITLE
Adds jit-grunt to project template

### LINKED ISSUE #47: Add jit-grunt to generated project
[jit-grunt](https://github.com/shootaroo/jit-grunt) is cool. We should be encouraging its use.

### PR DESCRIPTION
**Pull request by Scotchester**
Closes #47

### CODE REVIEW

File: app/templates/_package.json [hunk 1]
```diff
@@ -32,6 +32,7 @@
     "grunt-legacssy": "~0.4.0",
     "grunt-string-replace": "~1.0.0",
     "grunt-topdoc": "~0.3.0",
+    "jit-grunt": "~0.9.1",
     "load-grunt-tasks": "~1.0.0",
```
Comment (jimmynotjim) [inline app/templates/_package.json]: `load-grunt-tasks` should be removed as well right?

Comment (Scotchester) [inline app/templates/_package.json]: Probbbbably. I feel like there was a cf-component where we needed both. Let me see if I can dig that up.

Comment (Scotchester) [inline app/templates/_package.json]: Ah, that was specific to the use of cf-grunt-config. No need for it in the default dependencies here. I'll remove.

Comment (Scotchester) [inline app/templates/_package.json]: Done in e2e15fa

**Comment by jimmynotjim** [comment]
LGTM

**Comment by Scotchester** [comment]
I'd like to get one more :+1: before merging this.

**Comment by ascott1** [comment]
Here's another :+1:

**Comment by Scotchester** [comment]
TY!

**Pull request merged**